app/v2/controllers/V2MovementsController.scala

```diff
@@ -337,9 +338,12 @@ class V2MovementsControllerImpl @Inject() (
   }
 
   def attachLargeMessage(movementId: MovementId, messageId: MessageId): Action[JsValue] =
-    (authActionNewEnrolmentOnly andThen acceptHeaderActionProvider(acceptOnlyJson = true)).async(parse.json) {
+    Action.async(parse.json) {
```

Comment:
@drnaylor-oc We should still authenticate these requests right and only accept it if coming from upscan (what if a trader manages to guess the endpoint for example) - not sure what the best way to go about it is, perhaps a spike to investigate?

Reply:
If we can authenticate it, I'd want it, very much so -- if only to ensure we know we're talking to Upscan.

However, the trader won't be able to get here because it's not in the API definition. That definition controls the endpoints that the API platform proxy lets through. So, as long as we don't put it in the API definition, that's not a worry.

Reply:
Oh, that's good to know! Thanks :)